packages/common/src/scopeSupportFacets/gleam.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import {
+  LanguageScopeSupportFacetMap,
+  ScopeSupportFacetLevel,
+} from "./scopeSupportFacets.types";
+
+const { supported } = ScopeSupportFacetLevel;
+
+export const gleamScopeSupport: LanguageScopeSupportFacetMap = {
```

Comment:
I went through the list of all of these scopes support facets and chose all the ones that I think apply to gleam. Not all of them are implemented in the queries or in the tests yet though